package.json

```diff
@@ -17,5 +17,8 @@
     },
     "lint-staged": {
         "**/*": "prettier --write --ignore-unknown"
+    },
+    "dependencies": {
```

Comment:
This isn't the right place to add this; you need to add it to the `memory-viz/` subpackage.

I actually realized that this is also true for the `tmp` dev dependency for testing. Please go ahead and fix that in this PR (i.e., remove `tmp` from the dev dependency list above and move it to the `memory-viz/` subpackage.